src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -14,6 +14,7 @@
             [liberator.core :refer [defresource]]
             ;[liberator.dev :as liberator-dev]
```

Comment:
I guess since this PR didn't touch these and we're on the fence about keeping them in, I'd propose not to touch them for this PR.  I'm okay with @nwolfe filing a principle violation follow-up PR on this, if he'd like.